src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,143 @@
+package subway;
+
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import subway.dtos.request.LineRequest;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철노선 기능 테스트")
+@Sql("classpath:sql/truncate.sql")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+
+    private LineRequest lineRequest;
+
+    @BeforeEach
+    void setUp(){
+        lineRequest = new LineRequest("신분당선", "bg-red-600", 1L, 2L, 10L);
+        generateStations("왕십리역", "마장역");
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+
+    @DisplayName("지하철노선 생성")
+    @Test
+    void createLine(){
+
+        //when
+        LineApiTest.createLine(lineRequest);
```

Comment:
```suggestion
        createLine(lineRequest);
```

static import를 하면 이렇게 될 것 같네요.